client/src/overview/subComponents/ProductInfo/ProductCategory.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { useContext } from 'react';
+import Contexts from '../../../contexts/Contexts.tsx';
+
+export default function ProductCategory() {
+  const currentProductData = useContext(Contexts.ProductContext);
+
+  return (
+    <div className='productCatrgory'>
+      {currentProductData ? currentProductData.category : 'emptyProductCategory'}
```

Comment:
One possible way to write this line is:
`{currentProductData?. category || 'emptyProductCategory'}`

The `?.` operator will safely get a property, without having to check it first